splash/browser_tab.py

```diff
@@ -719,12 +745,14 @@ def log(self, message):
 
 class _BrowserTabLogger(object):
     """ This class logs various events that happen with QWebPage """
-    def __init__(self, uid, web_page, verbosity):
+    def __init__(self, uid, verbosity):
         self.uid = uid
-        self.web_page = web_page
         self.verbosity = verbosity
+        self.web_page = None
 
-    def enable(self):
+    def add_web_page(self, web_page):
```

Comment:
technically, nothing prevents one to add more pages, the only thing it does is connecting signals to their respective handlers.